.travis.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+###
+### Travis settings
+###
+sudo: required
+services:
+  - docker
+
+
+###
+### Env variables
+###
+env:
+  global:
+    - image=flaconi/ansible
+  matrix:
+    - TAG=23
+    - TAG=24
+    - TAG=25
+    - TAG=latest
+
+###
+### Install requirements
+###
+install:
+  # Get newer docker version
+  - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get update; then break; else i=$((i+1)); fi done
+  - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce; then break; else i=$((i+1)); fi done
+  - docker version
+
+
+###
+### Build Docker images
+###
+before_script:
+  - make update-base
+  - make build-${TAG}
+
+  # Test if it exists
+  - if ! docker images | grep "${image}" | grep "${TAG}"; then
```

Comment:
Travis seems to break on all numeric `TAG` values (`23`, `24` and `25`). Only the one which has `latest` as its value seems to work.
@danvaida Can u provide some help here.